### PR TITLE
Update AG Grid theme header icon alignment 

### PR DESCRIPTION
**Pull request by alycrys**
This was a request from MTK. Discussed and agreed with @bhoppers2008 

MTK has tested and they're happy

### CODE REVIEW

File: .changeset/strange-years-compare.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"@salt-ds/ag-grid-theme": patch
+---
+
+Update header icon alignment
```
Comment (origami-z) [inline .changeset/strange-years-compare.md:5]: Be more specific about the change? around left margin -> center align?

Comment (alycrys) [reply .changeset/strange-years-compare.md]: How does this sound?

**Review by feliciackh — APPROVED** [review]

**Comment by libertymayc** [comment]
is there a story to demo this?

**Comment by alycrys** [comment]
> is there a story to demo this?
There isn't. But you can check by applying a filter on any column and notice the filter icon position.

**Review by Fercas123 — APPROVED** [review]

**Pull request merged**